packages/m/movit/pspec_x86_64.xml

```diff
@@ -20,52 +21,7 @@
-            <Path fileType="data">/usr/share/movit/deinterlace_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/dither_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/fft_pass_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/flat_input.frag</Path>
-            <Path fileType="data">/usr/share/movit/footer.comp</Path>
-            <Path fileType="data">/usr/share/movit/footer.frag</Path>
-            <Path fileType="data">/usr/share/movit/gamma_compression_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/gamma_expansion_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/header.130.frag</Path>
-            <Path fileType="data">/usr/share/movit/header.150.frag</Path>
-            <Path fileType="data">/usr/share/movit/header.300es.frag</Path>
-            <Path fileType="data">/usr/share/movit/header.comp</Path>
-            <Path fileType="data">/usr/share/movit/highlight_cutoff_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/identity.frag</Path>
-            <Path fileType="data">/usr/share/movit/lift_gamma_gain_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/luma_mix_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/mirror_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/mix_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/multiply_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/overlay_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/overlay_matte_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/padding_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/resample_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/sandbox_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/saturation_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/slice_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/texture1d.130.frag</Path>
-            <Path fileType="data">/usr/share/movit/texture1d.150.frag</Path>
-            <Path fileType="data">/usr/share/movit/texture1d.300es.frag</Path>
-            <Path fileType="data">/usr/share/movit/vignette_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/vs.130.vert</Path>
-            <Path fileType="data">/usr/share/movit/vs.150.vert</Path>
-            <Path fileType="data">/usr/share/movit/vs.300es.vert</Path>
-            <Path fileType="data">/usr/share/movit/vs.vert</Path>
-            <Path fileType="data">/usr/share/movit/white_balance_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/ycbcr_422interleaved_input.frag</Path>
-            <Path fileType="data">/usr/share/movit/ycbcr_conversion_effect.frag</Path>
-            <Path fileType="data">/usr/share/movit/ycbcr_input.frag</Path>
```

Comment:
Are these files supposed to be gone? These looks like the filters/effects files.

Reply:
Yes this was changed in 1.7.0

https://git.sesse.net/?p=movit;a=blob;f=NEWS;h=e4f75616d782c4f7715afe6d61fda1345556609a;hb=dbe9f61e611fafb918328b6f99e4533d2401e775